arcgis-ios-sdk-samples/Maps/Read a geopackage/ReadGeopackageViewController.swift

```diff
@@ -45,32 +45,18 @@ class ReadGeopackageViewController: UIViewController, UIPopoverPresentationContr
                 rasterLayer.opacity = 0.55
                 return rasterLayer
             }
-
-            // Keep an array of all the feature layers and raster layers in this geopackage.
-            self?.allLayers.append(contentsOf: rasterLayers)
-            self?.allLayers.append(contentsOf: featureLayers)
-        }
-        
-        //add the source code button item to the right of navigation bar
-        (self.navigationItem.rightBarButtonItem as! SourceCodeBarButtonItem).filenames = [
-            "ReadGeopackageViewController",
-            "GPKGLayersViewController"
-        ]
-    }
-    
-    // MARK: - Segue to and from the Layer Control viewcontroller.
-    
-    override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
-        if let gpkgLayersVC = segue.destination as? GPKGLayersViewController {
-            // Provide the map and all layers to the layer controller UI.
-            gpkgLayersVC.map = mapView.map
-            gpkgLayersVC.allLayers = allLayers
-
-            gpkgLayersVC.popoverPresentationController?.delegate = self
+            
+            // Add the arrays of feature and reaster layers to the map.
```

Comment:
```suggestion
            // Add the arrays of feature and raster layers to the map.
```
Fix typo in comment.